aws/deep-learning-containers
# [TEST DRAFT] Disable everything except tf hvd cpu

*Issue #, if available:*
TF-2 EC2 Horovod CPU Test times out.

*Description:*
- N/A

*Tests run:*
- Disable all other tests and run only test_hvd_cpu --> Still stuck
- Set test timeout to 1020 (one of the last successful runs of this test from pr-166 had this particular test succeed in 688 seconds), and enable test output printout in ec2_training_test_executor.

*DLC image/dockerfile:*

*Additional context:*


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
